app/src/main/java/com/github/steroidteam/todolist/model/todo/Task.java

```diff
@@ -67,13 +68,15 @@ public Boolean isDone() {
         return done;
     }
 
-    public void setDone(Boolean done) {
+    public Task setDone(Boolean done) {
```

Comment:
I see that the returned task is used in the ViewModel.
However it looks a bit weird, maybe removeDueDate() could also return the Task for consistency?

Reply:
Yes, I returned the task itself so it was easier to reuse the new object: like `func(task.setDone(true)).`
But yes it is not consistent with all other methods.